mlir/lib/Quantum/Transforms/ChainedSelfInversePatterns.cpp

```diff
@@ -14,11 +14,15 @@
 
 #define DEBUG_TYPE "chained-self-inverse"
 
-#include "Quantum/IR/QuantumOps.h"
-#include "Quantum/Transforms/Patterns.h"
+#include "VerifyParentGateAnalysis.hpp"
```

Comment:
https://github.com/PennyLaneAI/guidance-docs/blob/master/development/cpp-dev-guidelines.md#include-file-use

I'd recommend reordering the imports here to adhere with the guidelines. Though, as to why clang-tidy/format doesn't catch this already is up for check.

Reply:
done